### ISSUE TITLE
Determine if we should have a "price explainer" page on the site or leave that to conversations

### ISSUE THREAD

**Issue by wslack** [cloud-gov/pages-core#1302]
If this is a good idea (and I'm not sure it is)

Highlight value proposition. What costs are saved by Federalist and static sites? What is included in the cost? Possibly offer a comparison table comparing features and capabilities between Federalist and its competitors.

Cost is mentioned only once in the documentation, and doesn’t describe what specifically a customer will get in return. From UX test: “What are other differences between the free version and paid version? Are there ads in the free version? How’s it different from github where I can make my own site, or wordpress?

**Comment by OpenGlobe** [comment]
I think this could be related to a gap analysis and some work around the IA - so I'd say we move those relevant tasks to a new issue for next sprint: 

- [ ] Identification in major gaps in content and artifacts

**Comment by OpenGlobe** [comment]
Let's also add a note to consider adding: 

  - [ ] Get started page
